wordpress-mobile/WordPress-iOS
# Extract notifications. Accessibility review

Add accessibility hints and labels that were lost while refactoring

To test:
- Check out the branch
- Activate VoiceOver
- Check that the action buttons for notifications provide labels and hints to voiceover
